client/src/pages/PostPage.js

```diff
@@ -1,17 +1,20 @@
 import React, { useEffect, useReducer } from "react";
+import { useHistory, useParams } from "react-router-dom";
 import axios from "axios";
 import { Modal } from "antd";
-import { useHistory, useParams } from "react-router-dom";
 
+// Local
+import Post, { CONTENT_LENGTH } from "components/Feed/Post";
 import EditPost from "components/CreatePost/EditPost";
+import { typeToTag } from "assets/data/formToPostMappings";
 import Loader from "components/Feed/StyledLoader";
 import { FEED } from "templates/RouteWithSubRoutes";
-import Post, { CONTENT_LENGTH } from "components/Feed/Post";
 import { StyledPostPage } from "components/Feed/StyledPostPage";
-import { typeToTag } from "assets/data/formToPostMappings";
 import { isAuthorOrg } from "pages/Feed";
-
 import { postReducer, postState } from "hooks/reducers/postReducers";
+
+// Constants
+import { FEED, LOGIN } from "templates/RouteWithSubRoutes";
```

Comment:
```suggestion
```

Remove this line and add `LOGIN` to earlier import from `"templates/RouteWithSubRoutes"`

Reply:
Done.